src/supertux/player_status.hpp

```diff
@@ -61,6 +61,7 @@ class PlayerStatus final
 
   std::string worldmap_sprite; /**< the sprite of Tux that should be used in worldmap */
   std::string last_worldmap; /**< the last played worldmap */
+  std::string last_worldmap_title; /**< the title of the last played worldmap */
 
```

Comment:
Marking this in case the worldmap data gets removed, this variable will be unnecessary.